File: Models/BasicStatistics.cs
using System;
using System.Net;

namespace Stratis.Dashboard.Models
{
	public class BasicStatistics
	{
		public long CurrentHeight { get; set; }
		public int ConnectedPeers { get; set; }
		public long MemPoolTransactionsCount { get; set; }
	}
}

File: ClientApp/src/app/models/basic-statistics.ts
export class BasicStatistics {
  currentHeight: number;
  connectedPeers: number;
  memPoolTransactionsCount: number;

  constructor() {
    this.connectedPeers = 0;
    this.currentHeight = 0;
    this.memPoolTransactionsCount = 0;
  }
}
